NitroxClient/GameLogic/EquipmentSlots.cs

```diff
@@ -24,6 +24,7 @@ public void BroadcastEquip(Pickupable pickupable, GameObject owner, string slot)
         {
             string ownerGuid = GuidHelper.GetGuid(owner);
             string itemGuid = GuidHelper.GetGuid(pickupable.gameObject);
+            pickupable.gameObject.transform.SetParent(null); // On Deserialized Function Try to find non-existent Parent set null to prevent that bug
```

Comment:
Are there any side effects related to doing this?  We are setting the parent to null but not re-assigning the parent at the end.  Could be dangerous.  

Another method that you may want to look into is using DeserializeIntoGameObject when we deserialize.  It has two parameters that may help: 'bool forceParent' and 'transform parent'.   That would allow us to not do this hack.

Reply:
if you dont set null on serialize on deserialize call try to find no found and return null its the same xD i only prevent error